docs/source-fabric/advanced/compile.rst

```diff
@@ -3,7 +3,7 @@ Speed up models by compiling them
 #################################
 
 Compiling your PyTorch model can result in significant speedups, especially on the latest generations of GPUs.
-This guide shows you how to apply ``torch.compile`` correctly in your code.
+This guide shows you how to apply `torch.compile <https://pytorch.org/docs/stable/generated/torch.compile.html>`_ correctly in your code.
```

Comment:
just thinking shall we for release replace all `stable` with the latest stable release at the time we build docs in `https://pytorch.org/docs/stable/...` just to prevent falling links for past docs, for example, our 2.0 referring PT 2.0 with stable link but not existing as stable with PT2.2...
note that nothing would change for build latest with master :)

cc: @lantiga